src/db/index.js

```diff
@@ -0,0 +1,35 @@
+import Sequelize from 'sequelize';
```

Comment:
This file `src/db/index.js` is no longer used in the project. It was replaced by `src/sequelize/models/index.js`. Please delete it.

Reply:
Resolved